images/Dockerfile.arktosvmruntime-base

```diff
@@ -0,0 +1,66 @@
+FROM ubuntu:18.04
+MAINTAINER Yunwen Bai <yunwen.bai@futurewei.com>
+
+# BUMP 23.11.2018
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN echo deb-src http://archive.ubuntu.com/ubuntu/ bionic main universe restricted >>/etc/apt/sources.list && \
+    echo deb-src http://archive.ubuntu.com/ubuntu/ bionic-updates main universe restricted >>/etc/apt/sources.list
+
+RUN apt-get -y update && apt-get -y upgrade && \
+    apt-get -y build-dep libguestfs && \
+    apt-get -y build-dep supermin && \
+    apt-get -y install apt-utils git libjansson-dev libhivex-ocaml-dev
+
+RUN apt-get install -y libguestfs-tools 
+
+# build libvirt 6.5.0
+RUN apt-get install -y build-essential libsasl2-dev libdevmapper-dev libgnutls28-dev \
+                       libxml2-dev e2fslibs-dev libcap-ng-dev libnl-3-dev \
+                       libselinux1-dev libssh2-1-dev libsystemd-dev \
+                       parted python2.7-dev libreadline-dev libxslt1-dev \
+                       libtirpc-dev libnl-route-3-dev libxml2-utils xsltproc \
+                       python-docutils docbook-xsl uuid-dev
+
+RUN curl -O https://libvirt.org/sources/libvirt-6.5.0.tar.xz && \
```

Comment:
Consider keeping a local copy of the tarball in repo to improve the reliability of the build process?

Reply:
this is ok here. note that we will push the base image so it won't need to be build so the build process will only build the most top layer of the image. --- this is the current virtlet build process procedure as well.